packages/component/src/Activity/Speak.js

```diff
@@ -38,26 +38,12 @@ const Speak = ({ activity, markAsSpoken, selectVoice, styleSet }) => {
 
   const lines = [speak || text];
 
-  attachments.forEach(({ content: { speak, subtitle, text, title } = {}, contentType }) => {
+  attachments.forEach(({ content: { speak } = {}, contentType }) => {
     switch (contentType) {
```

Comment:
Should this be changed to an if statement? Since we're only adding the speak attribute for AdaptiveCards, it seems a little excessive to have it as a switch.